src/modules/home/components/create-accommodation/create-accommodation.tsx

```diff
@@ -0,0 +1,104 @@
+import { useState } from 'react';
+import Button from '@mui/material/Button';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import Box from '@mui/material/Box';
+import { toast } from 'react-toastify';
+
+import { type AccommodationFormData } from '../../types/accommodation-form-data.interface';
+import { createAccommodationStyles } from './styles';
+import { AccommodationDetailsForm } from './accommodation-details-form.tsx';
+import { ImageUploader } from './image-uploader.tsx';
+import { useCreateAccommodation } from '#/modules/home/api/create-accommodation.ts';
+import { useAccommodationContext } from '#/modules/home/contexts';
+import { ENDPOINTS } from '#/modules/auth/constants';
+
+export function CreateAccommodation() {
+  const [activeStep, setActiveStep] = useState(0);
```

Comment:
If you're not planning to add more steps, limit the allowed numbers to 0 and 1. `useState<0 | 1>(0)`